apps/server/src/tasks/tasks.service.ts

```diff
@@ -19,6 +20,79 @@ import { Task } from './entities/task.entity';
 
 @Injectable()
 export class TasksService {
+  @Cron('* */1 * * * *')
+  triggerMarkingAsDone() {
+    this.markTasksAsDone();
+  }
+
+  async markTasksAsDone() {
+    const tasks: Task[] = await Task.find({
+      relations: ['chunks', 'chunkInfo'],
+    });
```

Comment:
We probably don't want to do that at one batch. This will exceed RAM even for a few tasks.

Reply:
And yes, I know we're doing it that way without limiting everywhere, but I thought it's a great point to introduce batching & limiting.